public/index.html

```diff
@@ -3044,7 +3045,7 @@ <h3 class="margin0">
                                 <span data-i18n="New">New</span>
                             </div>
                             <small data-i18n="or">or</small>
-                            <select id="world_editor_select" class="margin0">
+                            <select id="world_editor_select" class="widthNatural flex1 margin0">
```

Comment:
I know, I just, albeit crookedly, solved the problem for my language. I changed this place because the words in Russian in those places did not fully fit. I just didn't know how critical the change would be for you in these places. If there is another, nicer way to fix this, I'd be interested in seeing it.

Reply:
Okay, what about that list box that comes after the search string? Some words don't fit there either. Is it possible to edit it somehow so that it has a fixed length? Because I also applied the widthNatural flex1 margin0 parameter to it.

Reply:
Okay, I get it. In the next commit I will roll back the changes to these lines.